### PR TITLE
Finish README file of python graphene

### PR DESCRIPTION
**Pull request by Zesheng16**
- Finish README.md of python graphene.

### CODE REVIEW

File: ppml/trusted-big-data-ml/python/docker-graphene/README.md [hunk 1]
```diff
@@ -1,25 +1,97 @@
...
+
+  Next, you need to store the password you used for key generation, i.e., `generate-keys.sh`, in a secured file.
+
+  ```bash
+  sudo bash ../../../scripts/generate-password.sh used_password_when_generate_keys
+  ```
+
+
+
+## Run Native Python Examples
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/python/docker-graphene/README.md:53]: change to Run as spark local mode

File: ppml/trusted-big-data-ml/python/docker-graphene/README.md [hunk 2]
```diff
@@ -80,39 +164,165 @@ SGX=1 ./pal_loader bash -c "/opt/jdk8/bin/java \
...
+
+The result should be similar to
+
+>test accuracy 0.78357
+>
+>training data accuracy 0.92361
+
+
+
+## Run Spark Standalne Examples
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/python/docker-graphene/README.md:210]: to Run as Spark Standalone mode

Comment (Zesheng16) [reply ppml/trusted-big-data-ml/python/docker-graphene/README.md]: Has been changed.

File: ppml/trusted-big-data-ml/python/docker-graphene/README.md (hunk 1 above)
Comment (Zesheng16) [reply ppml/trusted-big-data-ml/python/docker-graphene/README.md]: Has been changed.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**